crates/host/src/wasmbus/host_config.rs

```diff
@@ -108,6 +114,11 @@ impl Default for Host {
             secrets_topic_prefix: None,
             version: env!("CARGO_PKG_VERSION").to_string(),
             max_execution_time: Duration::from_millis(10 * 60 * 1000),
+            // 10 MB
+            max_linear_memory: 10 * 1024 * 1024,
+            // 50 MB
+            max_component_size: 50 * 1024 * 1024,
+            max_components: 10000,
```

Comment:
These might be nice as constants somewhere

Reply:
Added!